### PR TITLE
tikv: update the default value of compaction-guard-min-output-file-si…

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

tikv: update the default value of compaction-guard-min-output-file-size from 8MB to 1MB

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/tikv/pull/14889

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1525,7 +1525,7 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 ### `compaction-guard-min-output-file-size`
 
 + 设置 compaction guard 启用时 SST 文件大小的最小值，防止 SST 文件过小。
-+ 默认值：8MB
++ 默认值：1MB
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1528]:
```suggestion
+ 默认值：：在 v7.2.0 之前版本中为 8MB，在 v7.2.0 及之后的版本中为 1MB。
```

**Review by qiancai — APPROVED** [review]
/lgtm

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1525,7 +1525,7 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 ### `compaction-guard-min-output-file-size`
 
 + 设置 compaction guard 启用时 SST 文件大小的最小值，防止 SST 文件过小。
-+ 默认值：8MB
++ 默认值：在 v7.2.0 之前版本中为 8MB，在 v7.2.0 及之后的版本中为 1MB。
```
Comment (qiancai) [inline tikv-configuration-file.md:1528]: 对于某些变量和配置项，可能不同版本的值不同。如果写 “在 v7.2.0 之前版本中为 8MB”，需要去每个有这个配置项的版本的文档里，看这个配置项的默认值是否为 8MB。

 如果觉得翻看麻烦，是否可以直接写 “默认值：从 v7.2.0 起，默认值为从 8MB 修改为 1MB 。”

Comment (Oreoxmt) [inline tikv-configuration-file.md:1528]:
```suggestion
+ 默认值：从 v7.2.0 起，默认值从 8MB 变更为 1MB。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -573,7 +573,7 @@ I/O rate limiter 相关的配置项。
 
 ### `retry-interval`
 
-+ 初始化 PD 连接时的重试间隔。
++ PD 连接时的重试间隔。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:576]:
```suggestion
+ 设置 PD 连接时的重试间隔。
```

**Review by qiancai — APPROVED** [review]
/lgtm

**Review by tonyxuqqi — APPROVED** [review]

**Pull request merged**